### DISCUSSION TITLE
Previous Builds are not shown

### DISCUSSION THREAD

**Discussion by danieltura** [codemagic-ci-cd/codemagic-docs#1433] [Q&A] [answered]
hello,
All my previous builds seem to disappear from codemagic history.
Please Help,

**Comment by himesh-cm** [comment, marked as answer]
Hi @danieltura, 

Personal plan: Build history and artifacts of apps on the personal account are stored in Codemagic for 30 days after which they are deleted.
Pay as you go Plan: 
Build history and artifacts of apps on the team account with Pay as you go pricing are stored in Codemagic for 60 days after which they are deleted. 

If this is not the case with you, please share your team id, so we can look into it.

**Reply by danieltura** [reply]
Thanks for the help
